Cargo.toml

```diff
@@ -13,7 +13,8 @@ exclude = ["etc/**/*", "examples/**/*", "tests/**/*", "Cargo.lock", "target/**/*
 [lib]
 
 [features]
-default = []
+default = ["battery"]
+battery = ["wgc/battery"]
```

Comment:
maybe it's easier to just add it to the feature list of near `features = ["raw-window-handle"]` ?
the default feature here doesn't exactly make sense for the Web target, where wgc is not even supposed to compile...
it would be good to check how this affect build times, too

Reply:
hmm, the problem with doing that is that you can't turn off the battery dep. I'm not sure if you can do different defaults on differnet platforms, will poke around a bit more.

Reply:
So in the implementation of #484 we learned that this should work fine as is, as cargo will just ignore the `wgc` feature if wgc is not a dep on that target.